cedar-policy-core/src/parser/text_to_cst.rs

```diff
@@ -1003,6 +1061,22 @@ mod tests {
             .expect("parser error")
             .node
             .expect("no data");
+
+        // In the tolerant AST we store the Policy Error node
+        #[cfg(feature = "tolerant-ast")]
+        assert_eq!(
+            policies
+                .clone()
+                .0
+                .into_iter()
+                .filter_map(|p| p.node)
+                .count(),
+            3
+        );
+        #[cfg(feature = "tolerant-ast")]
+        return ();
```

Comment:
instead of this, we could guard the last assert (below) with `cfg(not(feature = "tolerant-ast"))`

Reply:
Sure, will do